apps/wallet/src/ui/app/hooks/useQredoInfo.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { useQuery } from '@tanstack/react-query';
+
+import { useBackgroundClient } from './useBackgroundClient';
+
+export function useQredoInfo(qredoID?: string) {
+    const backgroundClient = useBackgroundClient();
+    return useQuery({
+        queryKey: ['qredo', 'info', qredoID],
+        queryFn: async () => backgroundClient.getQredoConnectionInfo(qredoID!),
+        enabled: !!qredoID,
+        staleTime: 0,
```

Comment:
I think `staleTime` is 0 by default so not sure if this is doing anything:

https://tkdodo.eu/blog/practical-react-query#the-defaults-explained

Reply:
We set other defaults [here](https://github.com/MystenLabs/sui/blob/main/apps/wallet/src/ui/app/helpers/queryClient.ts#L11) I think default staleTime is 30s